### PR TITLE
aktualizr: bump to latest for server URL newline fix.

### PR DESCRIPTION
**Pull request by patrickvacek**
Signed-off-by: Patrick Vacek <patrickvacek@gmail.com>

Passes oe-selftest.

### CODE REVIEW

**Comment by lbonn** [comment]
Did we forget to make the necessary changes for https://github.com/advancedtelematic/aktualizr/pull/973? (will be a bigger problem for stable branches)

**Comment by patrickvacek** [comment]
> Did we forget to make the necessary changes for [advancedtelematic/aktualizr#973](https://github.com/advancedtelematic/aktualizr/pull/973)?

Yes. My bad. Let me try again.

**Comment by lbonn** [comment]
It's not a new problem apparently, we already forgot when we bumped to 3c1c77c005fc1f872f1e12080528ed6f8a32bbf3.

**Comment by patrickvacek** [comment]
@lbonn I think I've fixed the problem now by using /etc/sota/ecus.

File: recipes-sota/aktualizr/environment.inc [hunk 1]
```diff
@@ -1,11 +1,8 @@
-export SOTA_VIRTUAL_SECONDARIES
-
 do_install_append() {
-  for sec in ${SOTA_VIRTUAL_SECONDARIES}; do
-    AKTUALIZR_PARAMETERS_VIRTUALSECS="${AKTUALIZR_PARAMETERS_VIRTUALSECS} --secondary-config $sec"
-  done
-
-  echo "AKTUALIZR_CMDLINE_PARAMETERS=${AKTUALIZR_PARAMETERS_VIRTUALSECS}" > ${D}${libdir}/sota/sota.env
+  if [ -n "${SOTA_SECONDARY_CONFIG_DIR}" ]; then
```
Comment (lbonn) [inline recipes-sota/aktualizr/environment.inc]: We can get rid of the environment trick now and just use regular configuration.

File: recipes-sota/aktualizr/aktualizr_git.bb [hunk 2]
```diff
@@ -69,6 +69,12 @@ do_install_append () {
     install -m 0644 ${WORKDIR}/aktualizr-secondary.service ${D}${systemd_unitdir}/system/aktualizr-secondary.service
     install -m 0700 -d ${D}${libdir}/sota/conf.d
     install -m 0700 -d ${D}${sysconfdir}/sota/conf.d
+
+    if [ -n "${SOTA_SECONDARY_CONFIG_DIR}" ] && [ -e "${SOTA_SECONDARY_CONFIG_DIR}" ]; then
```
Comment (lbonn) [inline recipes-sota/aktualizr/aktualizr_git.bb]: Could be helpful to display a warning if the variable is defined but not an actual directory. 

Also better: `-d` instead of `-e`.

**Comment by patrickvacek** [comment]
@lbonn Fixed all your concerns. I'm still running oe-selftest but I've confirmed that it works as expected in the "default" auto-prov case.

**Comment by patrickvacek** [comment]
oe-selftest passes. I'll start updating the other PRs as well unless there are further concerns.
